data/rulesets.js

```diff
@@ -372,16 +372,39 @@ exports.BattleFormats = {
 		},
 		onValidateTeam: function (team, format) {
 			let abilityTable = {};
+			let clones = {
+				airlock: ['cloud9'],
+				battlearmor: ['shellarmor'],
+				clearbody: ['whitesmoke'],
+				cloud9: ['airlock'],
+				dazzling: ['queenlymajesty'],
+				insomnia: ['vitalspirit'],
+				moldbreaker: ['teravolt', 'turboblaze'],
+				powerofalchemy: ['receiver'],
+				queenlymajesty: ['dazzling'],
+				receiver: ['powerofalchemy'],
+				shellarmor: ['battlearmor'],
+				teravolt: ['moldbreaker', 'turboblaze'],
+				turboblaze: ['moldbreaker', 'teravolt'],
+				vitalspirit: ['insomnia'],
+				whitesmoke: ['clearbody'],
+			};
```

Comment:
No, Multiscale and Shadow Shield are different since shadow shield isnt affected by Mold Breaker like abilities, while Multiscale is. Same for FMB and Clear Body. That cloud9 mistake was stupid though, will fix it.